plugins/wme-sitebuilder/wme-sitebuilder/Wizards/FirstTimeConfiguration.php

```diff
@@ -350,9 +375,257 @@ public function setTagline( $tagline ) {
 			return;
 		}
 
-		if ( ! update_option( 'blogdescription', $tagline ) ) {
-			$this->errors[] = [ self::FIELD_TAGLINE => __( 'Invalid Tagline', 'wme-sitebuilder' ) ];
+		$this->getData()->set( 'blogdescription', $tagline );
+
+		if ( update_option( 'blogdescription', $tagline ) ) {
+			return;
+		}
+
+		$this->errors[] = [ self::FIELD_TAGLINE => __( 'Unable to save the Site Description', 'wme-sitebuilder' ) ];
+	}
+
+	/**
+	 * Get the site industry.
+	 *
+	 * @return string
+	 */
+	public function getIndustry() {
+		if ( empty( $this->fields[ self::FIELD_INDUSTRY ] ) ) {
+			$this->fields[ self::FIELD_INDUSTRY ] = get_option( 'site_industry', '' );
+		}
+
+		return $this->fields[ self::FIELD_INDUSTRY ];
+	}
+
+	/**
+	 * Set the site industry.
+	 *
+	 * @param string $industry
+	 */
+	public function setIndustry( $industry ) {
+		$industry = filter_var( $industry, FILTER_SANITIZE_FULL_SPECIAL_CHARS );
+
+		if ( $industry === $this->getIndustry() ) {
+			return;
+		}
+
+		$this->getData()->set( 'industry', $industry );
+
+		if ( update_option( 'site_industry', $industry ) ) {
```

Comment:
Following up based on the call earlier so I don't forgot. Unless we need to match the option up with a built-in WordPress key, we don't need to manually update an option as this will just add in extra unused data.